### PR TITLE
Minor proofreading proposals for docs

### PR DESCRIPTION
**Pull request by masonmark**
Hello, this PR is just a somewhat random set of very trivial proposed changes that I compiled during my first read-through of the docs. 

Just minor stuff like apostrophies, capitalization, and adding optional commas and simplifying phrases for clarity and to help non-native English speakers.

Congrats on the 1.0! 🙂

### CODE REVIEW

File: docs/guides/debugging/application.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ tauri::Builder::default()
 
 ### Using the Inspector in Production
 
-By default the inspector is only enabled in development and debug builds unless you enable it with a Cargo feature.
+By default the inspector is only enabled for development and debug builds unless you enable it with a Cargo feature.
```
Comment (amrbashir) [inline docs/guides/debugging/application.md]: I feel like "in" conveys the meaning better, but I am not a native speaker so I am just double checking

**Review by amrbashir — CHANGES_REQUESTED** [review]
Thank you, this is great

File: docs/guides/debugging/application.md (hunk 1 above)
Comment (masonmark) [reply docs/guides/debugging/application.md]: 👍 Yeah, I think that is valid.

**Comment by lorenzolewis** [comment]
@amrbashir does this look good to you now?

**Review by amrbashir — APPROVED** [review]
Looks good, thanks again

**Pull request merged**